internal/server/v1/iam/handler_test.go

```diff
@@ -0,0 +1,325 @@
+package iam_test
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	shieldv1beta1 "buf.build/gen/go/gotocompany/proton/protocolbuffers/go/gotocompany/shield/v1beta1"
+	"github.com/go-chi/chi/v5"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/goto/dex/internal/server/reqctx"
+	"github.com/goto/dex/internal/server/v1/iam"
+	"github.com/goto/dex/internal/server/v1/iam/mocks"
+	shareMocks "github.com/goto/dex/mocks"
+	"github.com/goto/dex/pkg/errors"
+	"github.com/goto/dex/warden"
+)
+
+func TestHandler_teamList(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		dexTeamListResonses := `{             
+			 "teams": [
+					{
+					"name": "data_fabric",
+					"created_at": "2023-10-13T14:42:58+05:30",
+					"updated_at": "2023-10-13T14:42:58+05:30",
+					"owner_id": 433,
+					"parent_team_identifier": "2079834a-05c4-420d-bfc8-44b934adea9f",
+					"identifier": "b5aea046-dab3-4dac-b1ea-e1eef423226b",
+					"product_group_name": "data_engineering",
+					"product_group_id": "2079834a-05c4-420d-bfc8-44b934adea9f",
+					"labels": null,
+					"short_code": "T394"
+					}
+			]}`
+		wardenClient := mocks.NewWardenClient(t)
+		frozenTime := time.Unix(1697188378, 0)
```

Comment:
add `.UTC()` at the end and readjust the `created_at` / `updated_at` in the `dexTeamListResponses` etc. to match that UTC version of time.. on the git CI runners, the timezone maybe UTC or some other (on our laptops, it will be UTC+5:30)

Reply:
Fixed.